src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  max-width: $card-size;
+  width: 100%;
+  box-sizing: border-box;
+  
+  display: flex;
+  flex-direction: column;
+
+  background: $white-color;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+
+  &__container-image {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    object-fit: cover;
+    max-width: 160px;
+    height: 134px;
+
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+
+    margin-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__star-review {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__buy {
+    margin-top: 14px;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    display: flex;
+    justify-content: center;
+
+    color: $white-color;
+
+    padding: 12px 0;
+    width: 100%;
+
+    background-color: $button-blue-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    transition: background-color 0.3s, color 0.3s;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions